middlewares/validators/authValidator.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable newline-per-chained-call */
+// const emailExist = require('../../lib/utils/checkers/emailExistChecker');
+// const userModelManager = require('../../lib/modelManagers/usermodel');
+
+const errorMessage = (res, message) => res.status(400).json({
+  status: false,
+  message
+});
+
+/**
+   * @description check against errors on signup fields
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
```

Comment:
`next` is a function and not an object

https://expressjs.com/en/guide/writing-middleware.html

Reply:
thanks